avocado/core/job.py

```diff
@@ -285,6 +286,11 @@ def _make_test_suite(self, urls=None):
                      Optionally, a list of tests (each test a string).
         :returns: a test suite (a list of test factories)
         """
+        suite = [(None, {})]
```

Comment:
This seems to work, and the code changes are simple and direct.

Still, I wonder if you pursued a different approach, more aimed at the loaders.  What I mean is, the entity responsible for resolving a collection of test names into a test suite is the loader.  In this example, we bypass the loader by creating a fake test suite (which shouldn't be the job direct responsibility).

I don't see a perfect way to do this without some other refactors, but, maybe `TestLoaderProxy.load_plugins()` could detect that a remote runner will be used, and thus, not load *any* plugin.  Then, `Job._make_test_suite()` should transparently return an empty suite.  The idea is to put the special logic where it's most appropriate, and all other used APIs should work be fulfilling their intended use.  When make runners and loaders proper plugins, the place where this special logic will be located should be much more clear.

Please let me know what you think of this suggestion.